adminimize.php

```diff
@@ -870,7 +870,7 @@ function _mw_adminimize_set_metabox_cp_option() {
 
 	$user_roles                = _mw_adminimize_get_all_user_roles();
 	$_mw_adminimize_admin_head = '';
-	$metaboxes                 = '';
+	$metaboxes                 = [];
```

Comment:
We need support for php5.2 :( so that we must use `array()` instead of `[]`. PLease can you change that. Thanks a lot.